src/applications/clock/timer.tsx

```diff
@@ -0,0 +1,152 @@
+import React from "../../../jsx-compiler/jsx";
+
+export const Timer = () => {
+
+    let seconds : number | undefined
+    let minutes: number | undefined
+    let hours : number | undefined
+    let timerInterval : number
+
+    const startTimer = () => {
+        const secondsDiv = document.getElementById('secondsTimer')
+        const minutesDiv = document.getElementById('minutesTimer')
+        const hoursDiv = document.getElementById('hoursTimer')
+
+
+        if (hours === undefined || minutes === undefined || seconds === undefined) {
```

Comment:
Pour le timer tu le complique la vie a check les hours, les minutes et les seconds, j'aurais mis tout en second genre timestamp puis lors de l'affichage tu fais juste une petite fonction de convertion, par exemple : https://stackoverflow.com/questions/1322732/convert-seconds-to-hh-mm-ss-with-javascript

Reply:
Modifié